application/static/train-hifigan.html

```diff
@@ -2,6 +2,8 @@
 {% block content %}
     {% if cuda_enabled %}
     <p>Training a custom vocoder <i>may</i> improve the synthesis quality for a given voice.</p><br>
+<p>If you trained your model on collab you can also <a href="https://colab.research.google.com/drive/1O_m8mUov5JzdqDz2X2KGuLqTnDLiEgDG?usp=sharing" target="_blank">your custom HiFiGan remotely</a>, using your existing datasets.</p>
```

Comment:
I think you meant to say 'train your custom Hifi-gan remotely'.
Maybe this is better:

`<p><a href="https://colab.research.google.com/drive/1O_m8mUov5JzdqDz2X2KGuLqTnDLiEgDG?usp=sharing" target="_blank">Train your custom vocoder remotely</a></p>`

Please also indent correctly :)